spec/factories/services/npq/npq_application.rb

```diff
@@ -1,104 +0,0 @@
-# frozen_string_literal: true
-
-FactoryBot.define do
```

Comment:
maybe we can also cleanup `lib/tasks/oneoff/data_cleanup/cleanup_applications.rake`?

Reply:
also maybe we can cleanup `docs/spec/fixtures/api_spec.json` as it was mostly like a vcr style spec for when we removed NPQ references, which isn't needed anymore?